src/connectors/aws_collect.py

```diff
@@ -454,6 +455,34 @@
         ('data_resources', 'VARIANT'),
         ('exclude_management_event_sources', 'VARIANT'),
     ],
+    # https://docs.aws.amazon.com/cli/latest/reference/inspector/list-findings.html
+    'inspector_list_findings': [
+        ('finding_arns', 'VARIANT')
```

Comment:
I think these all require `recorded_at`, `account_id`, and `region` columns, as the others. You can copy-paste them from above.